design-toolkit/components/src/components/hero/styles.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { tv } from '@/lib/utils';
+import type { VariantProps } from 'tailwind-variants';
+
+export type HeroStyleVariants = VariantProps<typeof HeroStyles>;
+
+export const HeroStylesDefaults = {
+  compact: false,
+  theme: 'dark',
+} as const;
+
+export const HeroStyles = tv({
+  slots: {
+    container: 'rounded-large p-xl',
+    icon: '',
+    primary: 'mb-s text-header-xl',
+    secondary: 'text-header-l',
+  },
+  variants: {
+    compact: {
+      false: {
+        container: 'grid grid-cols-[auto] gap-4',
+        icon: 'mt-s mb-l size-xxl',
+        primary: '',
+        secondary: '',
+      },
+      true: {
+        container: 'grid grid-cols-[auto_1fr] grid-rows-1 items-start gap-4',
+        icon: 'col-start-1 row-start-1 mr-xl size-[1.25lh]',
```

Comment:
The smaller icon size is a size already defined in the Icon component, it's medium.

Reply:
The sizes available on `Icon` are much smaller than the Figma designs.